web/themes/contrib/civictheme/.storybook/preview.js

```diff
@@ -109,3 +162,28 @@ export const parameters = {
     },
   },
 };
+
+export const decorators = [decoratorStoryLayout, decoratorDocs];
```

Comment:
we also added custom decorators that can now be accessed in the Drupal theme's storybook